content/300-guides/200-deployment/101-traditional-deployments/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: 'Deploying applications with Prisma to traditional (PaaS) environments'
+metaTitle: 'Deploy Prisma apps to traditional (PaaS) environments'
+metaDescription: 'Learn how to deploy your Prisma-backed apps to PaaS environments like Heroku, Koyeb, or AWS EC2'
+tocDepth: 2
+---
+
+<TopBlock>
+
+If your application is deployed via a Platform-as-a-Service (PaaS) provider, whether containerized or not, it is a traditionally-deployed app. Common deployment examples include [Heroku](./deploying-to-heroku) and [Koyeb](./deploying-to-koyeb).
```

Comment:
I know we only list 2 PaaS there for now, but I think the traditional deployment is used in many more cases (that are already listed and mentioned in the overview). Not sure if this sub overview should focus on PaaS. Maybe it could just have a sub headline and explanation of what PaaS are? And then over tiem we add a more generic guide that is not provider specific?

Reply:
(These might primarily be for Google and SEO...)